CellProfiler_pipelines/Pipelines/NF1_analysis_cp_plugin.cppipe

```diff
@@ -65,7 +65,7 @@ Groups:[module_num:4|svn_version:'Unknown'|variable_revision_number:2|show_windo
     grouping metadata count:1
     Metadata category:None
 
-RunCellpose:[module_num:5|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+RunCellpose:[module_num:5|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:['The default is 15 for minimum size that is used in Cellpose function, but I have it turned off. Can be defaulted back if needed.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
Is it worth documenting somewhere why you decided to turn it off?

Reply:
I am going to hold off on that since this was something I figured out was an unnecessary change in IDR_stream and I already have it set to turn off here since I assumed it is turned off in the Cellpose GUI. This comment was mainly for me, and I don't know how to explain why I turned it off expect for I thought that was how the GUI did it.